straxen/plugins/acqmon_processing.py

```diff
@@ -0,0 +1,131 @@
+import strax
+import straxen
+import numba
+import numpy as np
+from numba import njit
+from immutabledict import immutabledict
+
+export, __all__ = strax.exporter()
+
+#### Veto hardware ####:
+# V1495 busy veto module: 
+# Generates a 25 ns NIM pulse whenever a veto begins and a 25 ns NIM signal when it ends. 
+# A new start signal can occur only after the previous busy instance ended.
+# 1ms (1e6 ns) - minimum busy veto length, or until the board clears its memory
+    
+# DDC10 High Energy Veto:
+# 10ms (1e7 ns) - fixed HE veto length in XENON1T DDC10, 
+# in XENONnT it will be calibrated based length of large S2 SE tails
+# The start/stop signals for the HEV are generated by the V1495 board
+  
+
+@strax.takes_config(
+    strax.Option('max_veto_gap', default = int(5e8),
+                 help = 'Maximum separation between veto stop and start pulses [ns]'),
+    strax.Option('channel_map', track = False, type = immutabledict, 
+                 help = 'frozendict mapping subdetector to (min, max)'
+                      'channel number.'),
+    strax.Option('run_start_time', type = float, track = False, 
+                 default = int(0), help='time of start run (s since unix epoch)')
+)
+
+@export
+class VetoIntervals(strax.OverlapWindowPlugin):
+    """ Find pairs of veto start and veto stop signals and the veto duration between them
+    busy_*  <= V1495 busy veto for tpc channels
+    bb_*    <= V1495 busy veto for high energy tpc channels
+    hev_*   <= DDC10 hardware high energy veto
+    """
+        
+    __version__ ='0.0.19'
+    depends_on = ('raw_records_aqmon') 
+    provides  = ('veto_intervals')
+    data_kind = ('veto_intervals')
+
+
+    aqmon_channel_names = ('sum_wf','m_veto_sync',
+                           'hev_stop', 'hev_start', 
+                           'bb_stop','bb_start',
+                           'busy_stop', 'busy_start')
+    
+    tmp_dtype = strax.time_fields + [(('Interval between start and stop in [ns]', 'interval'), np.int64)]
+
+    def infer_dtype(self):
+        dtype = [(('faux time [ns]', 'time'), np.int64)]
```

Comment:
It is a fake time to keep strax happy, as imho plugins require "time" and "endtime" as part of their data type to work and its based on the start of the veto interval. Darryl suggested to streamline the final data type, if I'll have time I'll do it. Otherwise for now it would have to stay like that.